snakemake.md

```diff
@@ -0,0 +1,223 @@
+Snakemake
+=========
+
+
+What is it?
+-----------
+`Snakemake` is a python based workflow management system that allows for whole analysis pipelines to be automatized. It lets you define workflows as a set of rules, that state how to produce one or more output files from one or more input files. Dependencies between rules will be automatically inferred on execution and independent rules will be evaluated in parallel. If the output files for a rule are newer than the input files it will not be executed again. Snakemake can directly use conda or docker for reproducible software environments.  
```

Comment:
Try to make each sentence one line for better diffs. Anyway:
> It lets you define workflows as a set of rules that state how to produce one or more output files from one or more input files.

(Had superfluous comma.)